src/renderer/inc/RenderEngineBase.hpp

```diff
@@ -21,39 +21,91 @@ Author(s):
 #pragma once
 namespace Microsoft::Console::Render
 {
+    struct BufferLineRenderData
+    {
+        IRenderData* pData;
+        const TextBuffer& buffer;

```

Comment:
Ooh. I am seriously curious how much it will cost to just copy this and break the lock... (versus sending the ref)